shared/src/main/scala/org/atnos/eff/Cache.scala

```diff
@@ -122,68 +121,68 @@ class ConcurrentWeakIdentityHashMap[K, V] extends ConcurrentMap[K, V] {
   private val queue = new ReferenceQueue[K]
 
   override def putIfAbsent(key: K, value: V): V = {
-    purgeKeys
+    purgeKeys()
```

Comment:
I'm not big on using `()` to signal unit value return but I know there are different views on the subject :-).